system/services/assetManager/NativeImageService.cfc

```diff
@@ -193,7 +194,24 @@ component displayname="Native Image Manipulation Service" {
 			}
 		}
 
-		pdf attributeCollection=pdfAttributes;
+		if ( SERVER.lucee.version.left(1) gt 4 ) {
+			tmpFilePDF  = GetTempDirectory() & imagePrefix & "_page_" & arguments.page & ".pdf";
+			tmpFileJPG  = GetTempDirectory() & imagePrefix & "1.jpg";
+			FileWrite( tmpFilePDF, pdfAttributes.source );
+
+			var returnfileprefix = GetTempDirectory() & imagePrefix;
+			var BufferedImage    = createObject("java","java.awt.image.BufferedImage");
+			var ImageWriter      = createObject("java","org.apache.pdfbox.util.PDFImageWriter");
+			var document         = createObject("java","org.apache.pdfbox.pdmodel.PDDocument").load("#tmpFilePDF#");
```

Comment:
little thing, avoid `"#someVariable#" if you are able to just do `someVariable` instead :) i.e.

```
...).load( tmpFile );
```